_modules/week-8.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Week 8 - How should Big Tech contribute to digital commons ?
+---
+
+Slides
+: **PDF**{: .label .label-purple }
+  : [Download](/assets/pdfs/Digital%20Commons%20and%20Public%20Policy%20-%20Week%201.pdf)
```

Comment:
We don't have the slides yet. Can you comment by adding at the beginning of line 7 `<!--` and `-->`  at it's end? That way it won't display.